.eslintrc

```diff
@@ -29,9 +29,13 @@
     "mozilla/import-globals-from": 1,
     "mozilla/this-top-level-scope": 1,
 
+    "react/jsx-boolean-value": [2, "always"],
+    "react/jsx-closing-bracket-location": [2, "after-props"],
+    "react/jsx-curly-spacing": [2, "never"],
     "react/jsx-equals-spacing": [2, "never"],
     "react/jsx-no-duplicate-props": 2,
     "react/jsx-no-undef": 2,
+    "react/jsx-space-before-closing": [2, "always"],
```

Comment:
Unrelated, but just saw this in Tofino Slack... https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-string-refs.md

> _Currently, two ways are supported by React to refer to components. The first one, providing a string identifier is considered legacy in the [official documentation](https://facebook.github.io/react/docs/more-about-refs.html). Referring to components by setting a property on the this object in the reference callback is preferred._

Adding `"react/no-string-refs": 2` rule to your .eslintrc in this branch gives me 38 errors:

```sh
/Users/pdehaan/dev/github/mozilla/activity-streams/content-src/components/SiteIcon/SiteIcon.js
  22:9   error  Using this.refs is deprecated                          react/no-string-refs
  22:49  error  Using this.refs is deprecated                          react/no-string-refs
  42:12  error  Using string literals in ref attributes is deprecated  react/no-string-refs
  43:12  error  Using string literals in ref attributes is deprecated  react/no-string-refs
  45:14  error  Using string literals in ref attributes is deprecated  react/no-string-refs
  53:15  error  Using string literals in ref attributes is deprecated  react/no-string-refs
  55:12  error  Using string literals in ref attributes is deprecated  react/no-string-refs

...
```

Not sure if we want to enable that rule or not. According to the [Refs to Components](https://facebook.github.io/react/docs/more-about-refs.html) docs:

> _**Note:** Although string refs are not deprecated, they are considered legacy, and will likely be deprecated at some point in the future. Callback refs are preferred._
>
> &mdash; via https://facebook.github.io/react/docs/more-about-refs.html#the-ref-string-attribute